backends/cutie/cutie_export.py

```diff
@@ -1,24 +1,24 @@
-# 
+#
```

Comment:
Not strictly related to this PR, but we should probably integrate some formatting tools to avoid this in the future.